reg/drone/service/gnss/Sources.0.1.uavcan

```diff
@@ -0,0 +1,37 @@
+# There are multiple satellite networks and other data sources which provide autonomous geo-spatial positioning
+# with global coverage. These flags are used for identifying which of those data sources
+# are the source of the location data provided by the GNSS sensor node.
+
+bool gps_l1             # 1575.42 MHz
+bool gps_l2             # 1227.60 MHz
+bool gps_l5             # 1176.45 MHz
+
+bool glonass_l1         # 1602 + k 0.5625 MHz, k in [-7,6]
+bool glonass_l2         # 1246 + k 0.4375 MHz, k in [-7,6]
+bool glonass_l3         # TBA
+
+bool galileo_e1         # 1575.42 MHz
+bool galileo_e5a        # 1176.45 MHz
+bool galileo_e5b        # 1207.14 MHz
+bool galileo_e6         # 1278.75 MHz
+
+bool beidou_b1          # 1531.098 MHz
+bool beidou_b2          # 1207.140 MHz
+
+bool sbas               # Satellite-Based Augmentation System (WAAS, EGNOS, GAGAN, QZSS, etc.)
+bool gbas               # Ground-Based Augmentation System, DGPS (IMES, etc.)
+
+bool rtk_base           # Real-Time Kinematics base station data
+
+bool imu                # Inertial Measurement Unit
+bool vo                 # Visual Odometry
```

Comment:
As I am a fervent believer in spelling out acronyms in code (especially those that can't be easily inferred) I suggest to replace `vo` with `visual_odometry`.

Reply:
This is a good catch because it actually goes against the recommendations given in the Spec [section 3.9.1]:

>  Short names, unnecessary abbreviations, and uncommon acronyms should be avoided